src/Games/NexusMods.Games.Larian/BaldursGate3/Diagnostics.cs

```diff
@@ -0,0 +1,35 @@
+using JetBrains.Annotations;
+using NexusMods.Abstractions.Diagnostics;
+using NexusMods.Abstractions.Diagnostics.References;
+using NexusMods.Abstractions.Diagnostics.Values;
+using NexusMods.Generators.Diagnostics;
+
+
+namespace NexusMods.Games.Larian.BaldursGate3;
+
+internal static partial class Diagnostics
+{
+    private const string Source = "NexusMods.Games.Larian.BaldursGate3";
+    
+    [DiagnosticTemplate]
+    [UsedImplicitly]
+    internal static IDiagnosticTemplate MissingDependencyTemplate = DiagnosticTemplateBuilder
+        .Start()
+        .WithId(new DiagnosticId(Source, number: 1))
+        .WithTitle("Missing required dependency")
+        .WithSeverity(DiagnosticSeverity.Warning)
+        .WithSummary("Mod {PakMod} is missing required dependency '{MissingDependencyModName}'.")
+        .WithDetails("""
+                     '{MissingDependencyModName}' is a required by '{PakModuleName}' but is not present in the loadout.
```

Comment:
"is a required by" is a typo,
`MissingDependencyModName` has the version number, but `PakModuleName` does not. I assume you'll change this when adding outdated dependency detection.

Reply:
`MissingDependencyModName` only happened to have a version number because that was part of the mod name. Will fix the typo, good catch. 

Yeah this will get cleaned up as I add outdated checks and missing but actually just disabled in the loadout etch checks

Reply:
Then the filename shouldn't be used to name the mod, the name of the mod as defined in the meta.lsx file does not have the version number, only the zip filename has it.